components/Leads.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components'
+import { NavLink } from 'react-router-dom'
+
+const Link = styled(NavLink)`
+  font-size: 20px;
+  margin-bottom: 10px;
+  text-align: center;
+`
+
+export default ({ slug, leagueSlug, name }) => (
+  <Link
```

Comment:
Having a link that just sends you to the page you are currently on is quite an odd choice